.eslintrc

```diff
@@ -24,7 +24,12 @@
     "no-console": 0,
     "no-alert": 0,
     "object-curly-spacing": 0,
-    "react/jsx-space-before-closing": 0
+    "react/jsx-space-before-closing": 0,
+    "arrow-body-style": 0,
+    "max-len": [1, 120],
+    "no-param-reassign": 0,
+    "consistent-return": 0,
+    "no-use-before-define": 0
```

Comment:
`arrow-body-style` is ridiculously ugly for HOCs: https://github.com/eslint/eslint/issues/4353